pymatgen/io/core.py

```diff
@@ -68,7 +68,6 @@ def write_file(self, filename: str | Path) -> None:
 
         Args:
             filename: The filename to output to, including path.
-            kwargs: Keyword arguments passed to get_string()
```

Comment:
Yes, I thought about it indeed. But mypy was really not happy about it, so in the end I decided to do it like this to avoid future developers having problems with mypy.